packages/itwinui-css/src/avatar/avatar.scss

```
@@ -3,59 +3,39 @@
 @use '../mixins';
 @use './mixins' as *;
 
+$avatar-text-color: hsl(0 0% 0% / 0.6); // same as badge, meets AA contrast against soft backgrounds
```

```suggestion
$avatar-text-color: hsl(0 0% 0% / 0.77); // same as badge, meets AA contrast against soft backgrounds
```
If we change the opacity from `0.6` to `0.77` we can pass AAA contrast against all soft backgrounds.  If we do this, I would suggest the same change to `badge.scss`.

---

updated in avatar for now. looks nice!

we could make this a shared variable in `@itwin/itwinui-variables`? 🤔

---

> we could make this a shared variable in `@itwin/itwinui-variables`? 🤔

Since this value is being used in both avatar & badge, yeah, I think that would be a good idea.

---

I think the last thing I was doing on Friday before quitting for the weekend was exploring adding a `$avatar-icon-color` that has a little less contrast than the text but still meeting AAA requirement.  I don't think I landed on a value yet tho.

---

lets tackle this in a separate PR